demo/scripts/utils/trustedHTMLHandler.ts

```diff
@@ -0,0 +1,13 @@
+import * as DOMPurify from 'dompurify';
+
+const policy = !!window.trustedTypes
+    ? window.trustedTypes.createPolicy('sanitizeHtml', {
+          createHTML: (input: string) => DOMPurify.sanitize(input), //
+      })
```

Comment:
This portion is unnecessary.
You can call `DOMPurify.sanitize(input, {RETURN_TRUSTED_TYPE: true})` and [DOMPurify would return TrustedHTML](https://github.com/cure53/DOMPurify#what-about-dompurify-and-trusted-types).
So, you don't have to create Trusted Type policy.

Reply:
Good to know.

This is code just for demo site, not in production. It can also show how to create a trusted policy if user doesn't want to use DOMPurify.

Reply:
Oh, okay :) Sounds good then :)